src/test/java/com/smartsheet/api/internal/ReportResourcesImplTest.java

```diff
@@ -0,0 +1,130 @@
+package com.smartsheet.api.internal;
+
+/*
+ * #[license]
+ * Smartsheet SDK for Java
+ * %%
+ * Copyright (C) 2014 Smartsheet
+ * %%
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *      http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ * %[license]
+ */
+
+import com.smartsheet.api.SmartsheetException;
+import com.smartsheet.api.internal.http.DefaultHttpClient;
+import com.smartsheet.api.models.*;
+import junit.framework.TestCase;
+import org.apache.commons.io.output.ByteArrayOutputStream;
+import org.junit.Before;
+import org.junit.Test;
+
+import java.io.File;
+import java.io.IOException;
+import java.nio.file.Files;
+import java.nio.file.Paths;
+import java.util.ArrayList;
+import java.util.Arrays;
+import java.util.EnumSet;
+import java.util.List;
+
+import static org.junit.Assert.assertEquals;
+import static org.junit.Assert.assertNotNull;
+import static org.junit.Assert.assertTrue;
+
+public class ReportResourcesImplTest extends ResourcesImplBase {
+
+    private ReportResourcesImpl reportResources;
+
+    @Before
+    public void setUp() throws Exception {
+        reportResources = new ReportResourcesImpl(new SmartsheetImpl("http://localhost:9090/2.0/",
+                "accessToken", new DefaultHttpClient(), serializer));
+
+    }
+
+    @Test
+    public void testGetReport() throws SmartsheetException, IOException {
+        server.setResponseBody(new File("src/test/resources/getReport.json"));
+        Report report = reportResources.getReport(4583173393803140L, EnumSet.of(ReportInclusion.ATTACHMENTS, ReportInclusion.DISCUSSIONS), 1,1);
+        assertEquals(report.getPermalink(), "https://app.smartsheet.com/b/home?lx=pWNSDH9itjBXxBzFmyf-5w");
+        assertTrue(report.getColumns().get(0).getVirtualId() == 4583173393803140L);
+    }
+
+    @Test
+    public void testSendSheet() throws Exception {
```

Comment:
This test method is missing an assert statement. Test that sendSheet is returning the expected response.